k8s/helm/templates/blueprint-mask-role.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+  {{- include "k8s-blueprint-operator.labels" . | nindent 4 }}
+  name: {{ include "k8s-blueprint-operator.name" . }}-blueprint-mask-role
+rules:
+  # issue permissions to read/update fields beyond the status
+  - apiGroups:
+      - k8s.cloudogu.com
+    resources:
+      - blueprintmasks
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - k8s.cloudogu.com
+    resources:
+      - blueprintmasks/status
+    verbs:
+      - get
+      - patch
+      - update
```

Comment:
As you removed the status, this is not necessary anymore and might even lead to errors